src/std/array.ab

```diff
@@ -26,3 +26,52 @@ pub fun includes(array, value) {
     let result = array_first_index(array, value)
     return result >= 0
 }
+
+/// Returns the first element in the array.
+pub fun first(array) {
+    return array[0]
+}
+
+/// Returns the last element in the array.
+pub fun last(array) {
+    let index = len(array) - 1
+    return array[index]
+}
+
+/// Removes an element at the index from the array; this will have no effect
+/// if the index is before the start or beyond the end of the array.
+pub fun remove_at(ref array: [], index: Num): Null {
+    let offset = index + 1
+    let length = len(array)
+    array = array[0..index] + array[offset..length]
+}
```

Comment:
> if the index is before the start

Is it talking about the negative indexes? Because the tests show that they work. Or is this talking about the situation when `-5` is used on an array with 4 elements? If that's the case, then maybe we can phrase it better?

Reply:
I meant that the array would be unchanged if the index is out of bounds. I've reworded the doc comments to make this clear.